.travis/before_install.sh

```diff
@@ -1,12 +1,5 @@
 #!/usr/bin/env bash
 
-# WARNING: DO NOT EDIT!
-#
-# This file was generated by plugin_template, and is managed by bootstrap.py. Please use
-# bootstrap.py to update this file.
-#
-# For more info visit https://github.com/pulp/plugin_template
-
```

Comment:
Why should this header be removed?
The reference to `bootstrap.py` is wrong though.

Reply:
this is not from the template, it is the CI config for `plugin_template` itself, and it has some differences from the one generated from template.